IPython/utils/warn.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import warnings
 
-warnings.warn("The module IPython.utils.warn is deprecated, use the standard warnings module instead", DeprecationWarning)
+warnings.warn("The module IPython.utils.warn is deprecated since IPython 4.0 and will be remove in 6.0, use the standard warnings module instead", DeprecationWarning)
```

Comment:
Removed. 

I still think t if you don't give a deadline to people they won't update.